acp/identity/identity.go

```diff
@@ -8,34 +8,48 @@
 // by the Apache License, Version 2.0, included in the file
 // licenses/APL.txt.
 
-/*
-Package identity provides defradb identity.
-*/
-
 package identity
 
-import "github.com/sourcenetwork/immutable"
+import (
+	cosmosSecp256k1 "github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	"github.com/cosmos/cosmos-sdk/types"
+	"github.com/decred/dcrd/dcrec/secp256k1/v4"
```

Comment:
Discussed in call: `github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1` will be removed when we switch to DIDs.

Reply:
> Discussed in call: `github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1` will be removed when we switch to DIDs.

But we still need it for the one case of the hardcoded source address so I think it will remain :/